environment.rb

```diff
@@ -0,0 +1,102 @@
+module Environment
+	class Depot
+		attr_accessor :packs
+		def initialize
+			@packs = {
+				:simple_transportation_pack => {
+					:intelligence => [:cellphone],
+					:items => [:medipack, :chevy_versa]						
+				},		
+				:standard_transportation_pack => {
+					:intelligence => [:cellphone, :antenna],
+					:arsenal => [:colt_1911],
+					:items => [:handcuffs, :medipack, :chemistry, :chevy_versa]
+				},
+				:simple_mission_pack => {
+					:intelligence => [:infopack, :laptop, :cellphone, :antenna],
+					:arsenal => [[:colt_1911], [:handcuffs, :medipack, :chemistry, :financial]]
+				},
+				:standar_mission_pack => {
+					:intelligence => [:infopack, :laptop, :cellphone, :antenna],
+					:arsenal => [:remington_870, :colt_1911, :machete, :hatchet],
+					:items => [:handcuffs, :medipack, :chemistry, :financial]
+				}
+			}
+		end
+    end
+
+    class Control
+		attr_accessor :missions
+		@@modes = [:paused, :aborted, :accomplished, :failed]
+		def initialize
+			@missions = {}
+		end
+
+		def new_mission(name:, objective:, pack:)
+			@missions[name] = {
+				:objective => objective,
+				:status => :active,
+				:pack => pack
+			}
+		end
+		
+		@@modes.each do |mode|
+			define_method("set_mission_to_#{mode}") do |mission|
+				@missions[mission][:status] = mode
+			end
+		end
+    end
+
+
+	class Human
+		attr_accessor :id, :name, :personal_data, :professional_data
+
+		def initialize(name:)
```

Comment:
Methods with parameters according to the convention is better to write them without parenthesis.